+incf/+livescripts/setup.m

```diff
@@ -0,0 +1,72 @@
+%% startup.m
+% Setup variables and automatically download data set from DANDI depending how and "where" the code is ran.
+%
+%% Inputs:
+%
+%    from_location  : the location from where the live-script is ran. Could be either "local", "dandi" or "online".
+%    dandiset_id    : the ID of the DANDI data set.
+%    nwb_path       : the path towards the NWB files within the DANDI data set (keyword argument).
+%    util_libs      : the list of paths relative to the live-script that uses this function that points towards any utils libraries.
+%
+%% Outputs:
+%
+%    dataPath       : the path where the NWB data will be accessible
+%
+%% Example:
+%
+%    nwb_final_path = setup("dandi", "000011", nwb_path="sub-255201_ses-20141124_behavior+ecephys+ogen.nwb", util_libs=["helper-functions"])
+%
+%  This call will setup the live-script to use the 000011 data set, considering that the live-script that requires this setup is running on DANDI-Hub.
+%  This call also setup the NWB file to be located in "000011/sub-255201_ses-20141124_behavior+ecephys+ogen.nwb"
+%  and loads in MATLAB the functions from the "helper-functions", located at the same level as the live-script calling this setup function.
+%
+function dataPath = setup(from_location, dandiset_id, varargin)
+    persistent p
+    if isempty(p)
+        p = inputParser;
+        addRequired(p, 'from_location', @(x)mustBeNonempty(validatestring(x, {'local', 'dandi', 'online'})))
+        addRequired(p, 'dandiset_id', @(x)validateattributes(x, {'string', 'char'}, {'nonempty'}))
+        addParameter(p, 'nwb_path', '', @(x)validateattributes(x, {'string', 'char'}, {'nonempty'}))
+        addParameter(p, 'util_libs', [])%, @(x)mustBeNonempty)
+
+    end
+    parse(p, from_location, dandiset_id, varargin{:});
+    nwb_path = p.Results.nwb_path;
+    util_libs = p.Results.util_libs;
+
+    % Add the helper functions
+    arrayfun(@(x) addpath(strcat(fileparts(matlab.desktop.editor.getActiveFilename), filesep, x)), util_libs)
+
+    switch from_location
+        case "online"
+            % TODO
```

Comment:
I think this can be the same as the "local" case. "local" works for MATLAB Online for me...

Reply:
So I think this case can be removed, unless "online" is meant as something else.

Reply:
This is meant to be [MATLAB Online](https://www.mathworks.com/products/matlab-online.html). For this case, the code needs to adapt/optimize to the conventions of MATLAB Drive. 

et's be sure to discuss this soon, as this is a faster & wider pathway to entering the NWB ecosystem for select examples.

Reply:
@rly Thanks a lot for the detailed comments and suggestion! I'll integrate all your modification to the PR.

@vijayiyer05 @rly Regarding the `"online"` case, what I can do at the moment is to integrate it with the `"local`" case to avoid having a choice made by the user that does not trigger any behavior. What do you think?

```matlab
swtich from_location
    case {"online", "local"}
...
```

Reply:
My suggestion would be to just return a friendly error message saying MATLAB Online support is not yet facilitated and that directs users to use Local instead and to manually create/select a folder in their MATLAB Drive.

Reply:
@vijayiyer05 I just updated the PR with the message as you suggested.